packages/flame_svg/lib/svg_component.dart

```diff
@@ -18,7 +18,11 @@ class SvgComponent extends PositionComponent {
     super.anchor,
     super.children,
     super.priority,
-  }) : _svg = svg;
+    Paint? paint,
+  }) : _svg = svg {
+    this.paint = paint ?? this.paint
+      ..filterQuality = FilterQuality.high;
```

Comment:
Why are you setting the filterquality here? `FilterQuality.high` will make most SVGs worse.

Reply:
does it need to be removed ?

Reply:
Yeah, remove it. I guess we should remove it from the `Svg` class too, but that can be done later.

Reply:
removed